stripe/src/main/java/com/stripe/android/view/AddPaymentMethodNetbankingView.kt

```diff
@@ -0,0 +1,62 @@
+package com.stripe.android.view
+
+import android.util.AttributeSet
+import androidx.fragment.app.FragmentActivity
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.stripe.android.R
+import com.stripe.android.databinding.BankListPaymentMethodBinding
+import com.stripe.android.model.PaymentMethodCreateParams
+
+internal class AddPaymentMethodNetbankingView @JvmOverloads internal constructor(
+    activity: FragmentActivity,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+) : AddPaymentMethodView(activity, attrs, defStyleAttr) {
+    private var selectedPosition : Int? = null
+
+    private val netbankingAdapter = AddPaymentMethodListAdapter(
+        ThemeConfig(activity),
+        items = NetbankingBank.values().asList() as List<Bank>,
```

Comment:
don't need to cast to `List<Bank>`

Reply:
`values()` returns an array of FpxBanks so it can't cast it to a `List` so it crashes at runtime

`java.lang.ClassCastException: com.stripe.android.view.FpxBank[] cannot be cast to java.util.List`

Reply:
Oh you should use `toList`, not `asList`